src/apps/chifra/pkg/cache/types.go

```diff
@@ -0,0 +1,25 @@
+package cache
+
+type unsignedLong = uint64
```

Comment:
Do we need this?

Reply:
It's not vital in any way. It just makes `type cString` definition look like the one in C++.

Reply:
Let's remove it.